src/utils/dao-errors.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
```

Comment:
not terribly happy about disabling eslint errors but:
* disabling `max-classes-per-file` means we don't have to write one file per error type (bleh)
* `class-methods-use-this` appears to literally break `Symbol.toStringTag` which is required for use with Chai in our tests